src/instrument.js

```diff
@@ -411,6 +411,10 @@ function unliftStore(liftedStore, liftReducer) {
  * Redux instrumentation store enhancer.
  */
 export default function instrument(monitorReducer = () => null, options = {}) {
+  if (options.maxAge && options.maxAge < 2) {
```

Comment:
Let’s change it to `if (options.maxAge != null && options.maxAge < 2)` to also warn for `0`. To disable the option, they can pass `null` or `undefined`.